pkg/reconciler/ingress/status.go

```diff
@@ -445,7 +447,8 @@ func (m *StatusProber) listGatewayURLsPerPods(gateway *v1alpha3.Gateway) (map[st
 				}
 
 				for _, addr := range sub.Addresses {
-					urlsPerPods[addr.IP] = append(urlsPerPods[addr.IP], fmt.Sprintf(urlTmpl, portNumber))
+					ingressPodAddress := fmt.Sprintf("%s:%d", addr.IP, portNumber)
```

Comment:
```suggestion
					ingressPodAddress := net.JoinHostPort(addr.IP, string(portNumber))
```

Reply:
Unfortunately int32 cannot be converted to string like `string(portNumber)`, so we need to use `strconv.Itoa()` or `fmt.Sprintf()` in `JoinHostPort()` . Perhaps using just ` fmt.Sprintf()` without `JoinHostPort()` is simpler.

Reply:
Is is in theory, but JoinHostPort handles ipv6 stuff and we use it uniformly everywhere else I think. I'd rather go for the Itoa version in this case.

Reply:
Oh, I see. Then I should use it. Updated now. Thank you.